package.json

```diff
@@ -79,6 +85,7 @@
     "istanbul": "^0.4.0",
     "mocha": "^3.0.0",
     "mockery": "^2.0.0",
+    "node-sass": "^4.5.3",
```

Comment:
Where is Sass actually used here?  Is it some magic from the section in `compile.js`?